plugins/Installation/lang/de.json

```diff
@@ -31,7 +31,7 @@
         "JSTracking_Intro": "Um Ihre Website mit Matomo zu überwachen, müssen Sie sicherstellen, dass ein zusätzlicher Quellcode auf jeder Seite Ihrer Website eingefügt wurde.",
         "LargePiwikInstances": "Hilfe für große Matomo-Installationen",
         "Legend": "Legende",
-        "LoadDataInfileRecommended": "Falls Ihr Matomo Server Websites mit erhöhtem Verkehrsaufkommen (z.B.: > 100.000 Seiten pro Monat) verarbeiten muss empfehlen wir dieses Problem zu beheben.",
+        "LoadDataInfileRecommended": "Falls Ihr Matomo Server Websites mit erhöhtem Verkehrsaufkommen (z.B.: > 100'000 Seiten pro Monat) verarbeiten muss, empfehlen wir dieses Problem zu beheben.",
```

Comment:
I have changed this to "100 000 Seiten" as space is most common as a seperator in German